ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/imageSingle.test.js

```diff
@@ -0,0 +1,175 @@
+import capitalize from 'lodash/capitalize';
```

Comment:
Forgot to mention a subtle distinction that caused problems in VM 1.0 for image CVE or node CVE
* https://lodash.com/docs/4.17.15#capitalize
    > Converts the first character of string to upper case **and the remaining** to lower case.
* https://lodash.com/docs/4.17.15#upperFirst
    > Converts the first character of string to upper case.